debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -431,7 +438,8 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
                     TOMBSTONES_ON_DELETE,
                     SOURCE_STRUCT_MAKER_VERSION,
                     Heartbeat.HEARTBEAT_INTERVAL,
-                    Heartbeat.HEARTBEAT_TOPICS_PREFIX)
+                    Heartbeat.HEARTBEAT_TOPICS_PREFIX,
+                    SIGNAL_DATA_COLLECTION)
```

Comment:
Should really all support it by default? I'd rather have expected only for the Oracle connector to add this?

Reply:
@gunnarmorling Yes, I'd really like to make it available for all by default. Even the default logging action could be useful for lal connecotrs.